packages/graph-engine/src/nodes/typography/baseFontSize.ts

```diff
@@ -103,6 +108,9 @@ export default class NodeDefinition extends Node {
     const xHeightPX = (xHeightMM / 25.4) * (ppi / pixelDensity);
     const fontSizePX = (1 * xHeightPX) / xHeightRatio;
 
-    this.setOutput("value", fontSizePX);
+    const shift = 10 ** precision;
```

Comment:
I'm unsure about this .We're constantly replicating this functionality and it feels like we should rather have it external to the node

Reply:
can do

Reply:
added precision util